docs/source/introduction.rst

```diff
@@ -114,13 +114,13 @@ manipulation and can:
 -----------------
 
 
-Powerful, flexible, and very simple to produce visualisations of field
-constructs are available with the `cfplot` package, that is installed
+Powerful and flexible visualisations of field
+constructs are available with the `cfplot` package that is installed
```

Comment:
Could we somehow retain the notion that cf-plot is very easy to use (e.g. not all plotting libraries allow you to produce a plot with a one-liner)

Reply:
Indeed, I was wondering this myself - I guess we want to advertise that cf-plot is easy to use without appearing to be making bold and/or unsubstantiated claims such as 'very easy to use'.

> e.g. not all plotting libraries allow you to produce a plot with a one-liner

True, maybe this is the crux to focus in on. Can we say something along the lines of "_user-friendly_" or (something to lead into this) "with minimal setup" (e.g:

_"The `cf-plot` package, designed to produce visualisations from field constructs in as few lines/calls as necessary and in a powerful and flexible manner, ..."_

) which might highlight that, indirectly (former) or directly (latter)?

Just realising also we should correct it to `cf-plot` (avec hyphen), the actual library name! I'll do that now...

Reply:
I think I prefer `Powerful, flexible, and user-friendly visualisations of field ...`. Your call.

Reply:
I am more than happy to go with 'user-friendly', a more concise choice :slightly_smiling_face: Great, I'll add that into both equivalent cases raised here, then mark this as resolved.